src/patterns/create-a-username/index.md.njk

```diff
@@ -30,13 +30,14 @@ You should do research to understand this group and have a plan for helping them
 
 You should only ask users to create their own custom usernames if your service contains user-generated content that requires attributing, for example, a blog with comments. Allowing custom usernames lets users hide their identity if they want to.
 
-User-generated usernames are harder to create and easier to forget than email&nbsp;addresses.
+User-generated usernames are harder to create and easier to forget than email addresses.
 
 You need to:
 
-- tell people whether their proposed username is unique
-- suggest unique ones for them, in some cases
-- make sure people can retrieve or reset their username
+- tell the user whether their proposed username is unique
+- suggest a unique username for them, in some cases
+- make sure the user can retrieve or reset their username
+- accept usernames caselessly - for example, if the actual username is ‘Mary@example.com’, the user can still sign in with ‘mary@example.com’
```

Comment:
Any idea for how to display 'Mary/mary@example.com' in ordinary text?

In the preview, they both display as hyperlinks. I then tried putting backticks around them, but that made them display in code formatting.

Reply:
I looked in the email address pattern and it used this odd trick of putting an HTML tag in the middle of it.

Not sure if 'caselessly' is a good word? Tried to break it down but not sure 'regardless' is quite plain language enough either. Only other alternative I can think of so far is: "ignore letter case when accepting usernames" 

```suggestion
- accept usernames regardless letter case — for example, if the actual username is ‘Mary<i></i>@example.com’, the user can still sign in with ‘mary<i></i>@example.com’
```

Reply:
'Regardless' is definitely better than 'caselessly', but yeah, ain't the plainest for sure. Your other alternative sounds great (imo). And cheers for sharing the trick with the HTML tag! 👍🏻 

```suggestion
- ignore letter case when accepting usernames - for example, if the actual username is ‘Mary<i></i>@example.com’, the user can still sign in with ‘mary<i></i>@example.com’
```